package.json

```diff
@@ -51,6 +52,8 @@
     "clear": "node node_modules/react-native/local-cli/cli.js start --reset-cache",
     "lint": "eslint src",
     "lint:staged": "lint-staged",
+    "relay": "relay-compiler --src ./src --schema data/schema.graphql",
+    "relay:watch": "relay-compiler --src ./src --schema data/schema.graphql --watch",
```

Comment:
You can do `"relay:watch": "yarn relay -- --watch"` to pass an argument to another yarn script